### ISSUE TITLE
E0054 on abstract modifier after a decorator

### ISSUE THREAD

**Issue by Faithfinder** [quick-lint/quick-lint-js#1194]
Paste into web version:

```ts
@InterfaceType({
    resolveType: questionnaireTypeResolver,
})
abstract class QuestionnaireInterface {
    @Field()
    id!: string;

    @Field()
    title!: string;

    @Field(() => AssessmentType)
    type!: AssessmentType;
}
```

**Comment by strager** [comment]
Thanks for reporting. I fixed this in Git commit 77965450f24da5e027579db689be59f483c8da42. I will ship the fix in version 3.2.0.

**Comment by strager** [comment]
Fixed released in [version 3.2.0](https://quick-lint-js.com/releases/#3.2.0).
